Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -6555,6 +6555,9 @@ void CClientGame::RestreamWorld()
     m_pManager->GetPedManager()->RestreamAllPeds();
     m_pManager->GetPickupManager()->RestreamAllPickups();
 
+    if (bRemoveBigBuildings == true)
+        g_pGame->GetStreaming()->RemoveBigBuildings();
+
```

Comment:
Is there any reason to make this operation optional?

Reply:
Restreaming LOD models looks ugly and that's why I make it optional.

Reply:
Can you show, please?

Reply:
[Here](https://youtu.be/14HhpsBaxYU)

Reply:
Looks ok for me